HW0/P6/P6B.py

```diff
@@ -16,16 +16,25 @@ def burnTime(t):
 
     # Use a variety of wait times
     ratio = []
-    wait_time = []
+    wait_time = [pow(10, -k) for k in xrange(7)]
 
     for t in wait_time:
         # Compute jobs serially and in parallel
         # Use time.time() to compute the elapsed time for each
-        serialTime = 1
-        parallelTime = 1
+        serialTime = 0.
+        parallelTime = 0.
+
+        s_start = time.time()
+        for count in xrange(N):
+            burnTime(t)
+        serialTime = time.time() - s_start
+
+        p_start = time.time()
+        pool.map( burnTime(t), range(N) )
```

Comment:
It seems to be system dependent; I'm still getting 16x and the same graph. When I output the order of the function calls, it sleeps 16 times in a row, and then they all wake up around the same time.